packages/core/shared/src/plugins/taskPlugin/task.ts

```diff
@@ -162,7 +167,10 @@ export class Task {
       await Promise.all(
         this.getInputs('output').map(async key => {
           const inputPromises = (this.inputs[key] as MagickReteInput[])
-            .filter(this.filterNextTasks.bind(this))
+            .filter(con => {
+              this.filterOutputConnections.bind(this)
```

Comment:
Seems to work. The bund was necessary because we access this inside the bound function. Probably more elegant to just use .call instead and pass this in that way.